service_container.rst

```diff
@@ -623,6 +623,12 @@ But, you can control this and pass in a different logger:
 This tells the container that the ``$logger`` argument to ``__construct`` should use
 service whose id is ``monolog.logger.request``.
 
+.. tip::
```

Comment:
maybe `note` instead of `tip`, wdyt?

Reply:
I just copied the code from the Symfony 3.4 documentation where it ts a tip. Please check this out.
https://symfony.com/doc/3.4/service_container.html
and this. https://nimb.ws/ZQFxdu

`note` also works by the way.